typescript-json-schema.ts

```diff
@@ -372,7 +380,7 @@ export class JsonSchemaGenerator {
             }
         };
 
-        enm.members.forEach(member => {
+        members.forEach(member => {
```

Comment:
why not just `(node.kind === ts.SyntaxKind.EnumMember ? [enm] : enm.members).forEach(...)`?